### ISSUE TITLE
ZT interface missing on Raspbian after editing local.conf; can't bring it back

### ISSUE THREAD

**Issue by cferrey** [zerotier/ZeroTierOne#916]
**Describe the bug**
ZeroTier interface (ztc3q53d7l) is missing on Raspbian after editing flow rules and local.conf, and reinstalling Zerotier and/or rebooting the Raspberry Pi doesn't bring it back.

**To Reproduce**
1. Install Zerotier, join network
2. Add flow rule to network (drop dport 9993;).  This was in an attempt to fix issue #779.
3. Remove flow rule (didn't work to fix issue).
4. Add the following to local.conf in effort to fix issue: { "physical": { "NETWORK/bits": { 10.0.0.0/16 "blacklist": true } } }
5. Undo step 4, as it also didn't fix the issue.
6. Reboot the Raspberry Pi.
7. Run listepeers and receive the following error:
"Error connecting to the ZeroTier service: connection failed
Please check that the service is running and that TCP port 9993 can be contacted via 127.0.0.1."
8. Run ifconfig and see that the ZT interface is missing.  Running ifdown/ifup doesn't bring it back, nor does rebooting or uninstalling+reinstalling ZT.

**Expected behavior**
It's unclear to me how the ZT interface would have gotten permanently deleted by changing settings in local.conf.  I've undone all the changes I made and have uninstalled and reinstalled ZT in an effort to fix this, but nothing has worked.  This happened simultaneously on two separate Raspberry Pis on which I was performing the same set of steps from above.

**Desktop (please complete the following information):**
 - OS: Raspbian
 - ZeroTier Version: 1.2.12
 - Hardware: Raspberry Pi 3 B+
 
**Additional context**
Using Raspberry Pis as L2 bridging devices.  /etc/network/interfaces is configured as below and everything is on a 10.0.0.0/16 subnet both locally and in ZT.  ifconfig -a shows only br0, eth0, lo, and wlan0.

auto eth0
iface eth0 inet manual
iface eth0 inet6 manual

auto br0
iface br0 inet static
        address 10.0.3.2
        netmask 255.255.0.0
        gateway 10.0.3.1
        dns-nameservers 127.0.0.1
        bridge_ports eth0 ztc3q53d7l
        bridge_stp on

**Comment by laduke** [comment]
zerotier-one creates/deletes interfaces when it joins/leaves a network. 

` Please check that the service is running and that TCP port 9993 can be contacted via 127.0.0.1."`
Means: Usually means zerotier-one isn't running (check systemd) or you need to use sudo with zerotier-cli; sometimes it means iptables/ufw is blocking port 9993 to localhost. 

In my experience, when you're doing a single port bridge, you don't want zerotier to assign an IP, just the bridge should get one, like you have above^
There's 3 ways to stop it: the checkbox on the member in my.zerotier.com, `zerotier-cli set <networkId> allowManaged=0`, and editing /var/lib/zerotier-one/networks.d/<networkid>.local.conf

**Comment by glimberg** [comment]
Sounds like ZeroTier isn't set to run on startup. If ZeroTier isn't running, the interface won't exist.

**Comment by cferrey** [comment]
Thanks for the feedback.  Apologies for the stupid question, but when we talk about ensuring ZT is "running" -- we're talking about the zerotier-one service, right?  I've done `sudo service start zerotier-one` and have tried leaving/re-joining the network, but I keep getting the "check if running" error.

It's not such a terrible inconvenience to reinstall the OS, so I may just go that route -- I don't want to take focus away from more widespread issues. 

I do have one question about something laduke said: "In my experience, when you're doing a single port bridge, **you don't want zerotier to assign an IP,** just the bridge should get one, like you have above in /etc/network."  Does that mean I should not be giving the bridge devices any IP addresses in my.zerotier.com?  I figured I had to in order for the bridge to work.  I'm currently using the same virtual and local IP for the bridges, given that everything is in the same subnet.
